section4/exercises/person.rb

```diff
@@ -1,5 +1,30 @@
-# Create a person class with at least 2 attributes and 2 behaviors. 
+# Create a person class with at least 2 attributes and 2 behaviors.
 # Call all person methods below the class and print results
 # to the terminal that show the methods in action.
 
-# YOUR CODE HERE
+class Person
+  attr_accessor :name, :occupation, :hometown
+
+  def initialize(name, occupation, hometown)
+    @name = name
+    @occupation = occupation
+    @hometown = hometown
+  end
+
+  def promotion(new_position)
```

Comment:
You'll want method names to be verbs - so maybe instead of promotion, use `promote`.

`moved_towns` is a bit better, but I'd still recommend changing to `move_towns`. 

It sounds super nit picky but you always want your code to feel as easy as possible for others to read, and the way we do it is by following those conventions consistently!

Reply:
Makes sense and will do moving forward - thanks!